project/client/js/google-auth.js

```diff
@@ -0,0 +1,42 @@
+var googleUser = {};
+var initGoogleAuth = function() {
+    gapi.load('auth2', function(){
+        // Retrieve the singleton for the GoogleAuth library and set up the client.
+        auth2 = gapi.auth2.init({
+            client_id: '467471910617-vo4g6ojobn4o2tu25vo38ofurfis0jqj.apps.googleusercontent.com',
```

Comment:
That is the client ID uniquely identifying the app to Google's API. In order to make Google OAuth possible, you'll need to generate a pair of unique IDs for play.benbru.com: one of them will be this, which can be shared publicly, and the other one is the secret ID which can't be made public.

Once you've obtained those, add them to `app_config.tmpl` as with Facebook's:
```
# google credentials
google_secret = <SECRET>
google_app_id = <APP_ID>
```
I tried using Tera to fill in the ID straight from the config file by passing it as a parameter to the `initGoogleAuth()` function but this required those keys to be fed to every controller in order to support logout on any page.